### PR TITLE
doc: add team members to README.md

### PR DESCRIPTION
**Pull request by ghost**
##### Affected core subsystem(s)
doc

please comment or push your pronouns/other data if you can't find yours listed! also sorry if i got any wrong

### CODE REVIEW

**Review by TimothyGu — APPROVED** [review]

**Review by addaleax — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -133,10 +132,48 @@ us a report nonetheless.
...
+**Alexey Orlenko** &lt;eaglexrlnk@gmail.com&gt; (he/him)
+- [Harrison-M](https://github.com/Harrison-M) -
+**Harrison Massey** &lt;harrison.massey@gmail.com&gt;
+- [pup](https://github.com/pup) -
+**Olivia Hugger** &lt;olivia@fastmail.com&gt; (she/her)
+- [Qard](https://github.com/Qard) -
+**Stephen Belanger** &lt;admin@stephenbelanger.com&gt;
+- [sandfox](https://github.com/sandfox) -
+**James Butler** &lt;james.butler@sandfox.co.uk&gt;
+- [TimothyGu](https://github.com/TimothyGu] -
```
Comment (hiroppy) [inline README.md]: Syntax seems to be broken. 


```diff
- - [TimothyGu](https://github.com/TimothyGu] -
+ - [TimothyGu](https://github.com/TimothyGu) -
```

**Review by AgentAntelope — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (ghost) [reply README.md]: whoops

Comment (TimothyGu) [reply README.md]: fixed

**Review by varjmes — APPROVED** [review]

**Review by stefee — APPROVED** [review]

**Comment by SomeHats** [comment]
Re: https://github.com/ayojs/ayo/pull/69#discussion_r139661960, do we want to include discord usernames here?

**Comment by addaleax** [comment]
Hey everyone, I already mentioned it in the Discord, but: *please* add `[ci skip]` to commit messages that you push for documentation-only commits – otherwise we’re just hogging Travis resources & building up a nice backlog for our CI :)

**Comment by Harrison-M** [comment]
@addaleax oh whoops, sorry! I thought "I need to add [ci skip] to this" as i went to make the commit, then when I finished, I proceeded to open-palm slam that commit button without doing the thing. 😣

**Comment by addaleax** [comment]
@harrison-m yeah, no problem, that was more of a reminder for everyone else to follow :) it’s not like stopping CI tasks is a huge deal or anything ^^

**Comment by ghost** [comment]
@SomeHats i think i'd like to keep only email in here, as it's fairly easy to change your entire discord username, and we don't know whether we'll keep on using discord forever

**Comment by ljharb** [comment]
@addaleax you may be able to configure your travis.yml to do this automatically

**Review by aqrln — APPROVED** [review]
LGTM, thanks!

Small typographical nit: maybe use em-dashes?

**Comment by addaleax** [comment]
After chatting a bit with some of us in the Discord channel, I turned the lists into tables + added github icons … if you don’t like this, feel free to say so/just remove the commits here :)

Edit: Rendered: https://github.com/ayojs/ayo/blob/olivia/add-team-members/README.md#current-project-team-members

File: README.md [hunk 2]
```diff
@@ -133,10 +132,36 @@ us a report nonetheless.
...
+| ![TimothyGu](https://avatars1.githubusercontent.com/u/1538624?v=4&s=64) | [TimothyGu](https://github.com/TimothyGu) | **Timothy Gu** | timothygu99@gmail.com | he/him |
+| ![varjmes](https://avatars0.githubusercontent.com/u/542140?v=4&s=64) | [varjmes](https://github.com/varjmes) |  **James Spencer** | hello@jmes.tech | they/them |
+
+### Moderation Team
+
+| | Github | Name | Email | Pronouns |
+| ---| --- | --- | --- | --- |
+| ![AgentAntelope](https://avatars1.githubusercontent.com/u/1962412?v=4&s=64) | [AgentAntelope](https://github.com/AgentAntelope) | **Fell Sunderland** | agentantelope+github@gmail.com | he/him |
+| ![janl](https://avatars1.githubusercontent.com/u/11321?v=4&s=64) | [janl](https://github.com/janl) | **Jan Lehnardt** | jan@apache.org |
+| ![NoahDragon](https://avatars3.githubusercontent.com/u/5704064?v=4&s=64) | [NoahDragon](https://github.com/NoahDragon) | **Abner Chou** | hi@abnerchou.me | |
```
Comment (NoahDragon) [inline README.md]: The pronoun for me could be `he/him`.

**Comment by Qard** [comment]
he/him or they/them for me, thanks.

**Review by aqrln — APPROVED** [review]

**Comment by ghost** [comment]
landed in 14f86c8d158815022640acce086058314b872884!

**Pull request closed without merging**